internal/webhook/handler.go

```
@@ -0,0 +1,49 @@
+		http.Error(w, "Failed to retrieve Discord webhook URL", http.StatusInternalServerError)
+		log.Fatal().Msg("Discord webhook URL is not set")
+	}
+
+	if err := discord.SendMessage(webhookURL, payload); err != nil {
+		log.Error().Err(err).Msg("Failed to send Discord webhook message")
+		http.Error(w, "Failed to send Discord webhook message", http.StatusInternalServerError)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+	w.Write([]byte("Webhook received"))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)